### PR TITLE
<Skeleton/> - add uni.driver + migrate to stylable

### PR DESCRIPTION
**Pull request by EgoziE**
Adding uni.driver and migrating to stylable.

This PR was created to help estimating the effort of removing all scss files and creating uni.drivers for all components.

My estimation: small effort - big scope
If we split the work we can finish this in no time.

### CODE REVIEW

File: src/Skeleton/Skeleton.js [hunk 1]
```diff
@@ -1,28 +1,33 @@
...
+      >
+        {content.map(({ type, size }, key) => (
           <div
             key={key}
-            data-hook="placeholder-line"
-            className={classnames(styles.placeholderLine, styles[spacing], {
-              [styles.middle]: alignment === 'middle',
-            })}
+            data-hook={dataHooks.line}
+            {...styles('line', { alignment, spacing }, this.props)}
```
Comment (TalZa1810) [inline src/Skeleton/Skeleton.js:24]: why not using `styles.line`? you can get the `alignment` & `spacing` props from the root.

File: src/Skeleton/Skeleton.js [hunk 2]
```diff
@@ -1,28 +1,33 @@
...
           >
             <div
-              data-hook="placeholder-chunk"
-              className={classnames(styles.chunk, styles[item.size])}
+              data-hook={dataHooks.chunk}
+              data-type={type}
+              data-size={size}
+              {...styles('chunk', { size }, this.props)}
```
Comment (TalZa1810) [inline src/Skeleton/Skeleton.js:30]: same comment here. just pass the `size` prop to the root div

File: src/Skeleton/Skeleton.spec.js [hunk 3]
```diff
@@ -11,32 +16,44 @@ const content = [
...
-      expect(driver.hasSpacing('medium')).toBe(true);
-    });
+describe('[async]', () => {
+  runTests(createRendererWithUniDriver(skeletonUniDriverFactory));
+});
 
-    it('should have lines with expected sizes', () => {
-      expect(driver.hasSizes(content.map(({ size }) => size))).toBe(true);
+function runTests(render) {
+  afterEach(() => cleanup());
```
Comment (TalZa1810) [inline src/Skeleton/Skeleton.spec.js:28]:
```suggestion
  afterEach(cleanup);
```

File: src/Skeleton/Skeleton.js (hunk 1 above)
Comment (EgoziE) [reply src/Skeleton/Skeleton.js:24]: Alignment and spacing is a property relevant to the line only, it has no effect it the root scope.

In the css itself
on root:
`.root:alignmet(sheker) .line { ... }`
on line:
`.line:alignmet(sheker){ ... }`

**Pull request merged**